### DISCUSSION TITLE
Primeiro passo

### DISCUSSION THREAD

**Discussion by Alice-74** [Alice-74/Alice-74#2] [Q&A] [answered]
Como que cria um repositorio

**Comment by ghost** [comment, marked as answer]
E só ir na aba repositório em new, colocar um nome que não existe no seu perfil, add um readm e bem útil em publico e por fim criar

**Comment by ghost** [comment]
E só ir na aba repositório em new, colocar um nome que não existe no seu perfil, add um readm e bem útil em publico e por fim criar
